### DISCUSSION TITLE
Increase Sampling frequency?

### DISCUSSION THREAD

**Discussion by Leonardo-Ferreira** [open-telemetry/opentelemetry-dotnet#5582] [Q&A]
Metrics such as "process.memory.usage" are only collected once per minute. Is there a way to sample this like, once every 15 sec? same thing for other metrics such as HTTP queue, open_connections and others

**Comment by cijothomas** [comment]
Currently, OpenTelemetry (OTel) does not have a separate concept of "collection/sampling frequency." The collection frequency is inherently tied to the export interval, which defaults to 60 seconds for the OTLP Exporter. You can modify this interval to change the frequency at which metrics are exported. This adjustment will automatically affect the sampling/collection frequency for Observable instruments!

There have been [spec discussions](https://github.com/open-telemetry/opentelemetry-specification/issues/1432) about introducing a distinct sampling frequency, independent of the exporter frequency in the OTel specification. However, this proposal has not materialized as of today.
